go/consensus/api/api.go

```diff
@@ -110,6 +110,9 @@ type ClientBackend interface {
 	// GetEpoch returns the current epoch.
 	GetEpoch(ctx context.Context, height int64) (epochtime.EpochTime, error)
 
+	// EpochtimeConsensusParameters returns the epochtime consensus parameters.
```

Comment:
Maybe file an issue to remove this when the new beacon backend is in place? Or mention it in #3180.

Reply:
will comment on #3180 after i merge this